presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorCommitHandle.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spi.connector;
+
+import com.facebook.presto.spi.ConnectorId;
+
+import java.util.List;
+import java.util.Set;
+
+import static java.util.Collections.emptyList;
+import static java.util.Collections.emptySet;
+
+public interface ConnectorCommitHandle
+{
+    default List<?> getOutputLastDataCommitTimes(String outputKey)
+    {
+        return emptyList();
+    }
+
+    default boolean containOutput(String outputKey)
+    {
+        return false;
+    }
+
+    default void removeOutput(String outputKey)
+    {
+    }
+
+    default Set<String> getOutputKeys()
+    {
+        return emptySet();
+    }
+
+    default ConnectorId getConnectorId()
+    {
+        return new ConnectorId("Unknown");
+    }
```

Comment:
remove all the methods in this class; check my other comments in `SemiTransactionalHiveMetastore` and `HiveCommitHandle`

Reply:
I understand that we should not mutate the content of a handle; should we keep use getter functions?

Reply:
We can also remove all getter functions. They are not used

Reply:
They are actually used in the following PRs. I will remove them for now.